src/image_tools/preflight.py

```diff
@@ -71,6 +71,14 @@ def preflight_commands(images: List[str], args: Namespace, conf) -> Dict[str, Co
                     f"ospid-{conf.open_shift_projects[args.product]['id']}",
                  ]
             )
+        if args.architecture:
+            cmd_args.extend(
+                ["--platform",
+                    # this argument value has already been checked against valid values with an expected prefix.
+                    # Preflight provides the same "linux/" prefix and so it must be removed here.
+                    args.architecture.split("linux/")[1],
```

Comment:
a slightly more pedantic version if you want:
```suggestion
                    args.architecture.split("/")[-1],
```

Reply:
I left it specific on purpose, as I think it would be good to know if any other prefixes suddenly start being used that we haven't tested with preflight.

Reply:
they would not reach this point as the `check_architecture_input() ` would reject them.

I'll approve anyway